docs/advanced/additional-requests.rst

```diff
@@ -919,3 +889,118 @@ an :class:`~.HttpClient` instance:
 From the code sample above, we can see that every time an :class:`~.HttpClient`
 is created for Page Objects needing an ``http_client``, the specific **request
 implementation** from a given framework is injected to it.
+
+Downloader Behavior
+-------------------
+
+The Downloader should be able to accept an instance of :class:`~.HttpRequest`
```

Comment:
It's a general comment about framework docs. I wonder if we should use stronger language, similar to how it's used in RFCs: https://www.ietf.org/rfc/rfc2119.txt. 

So, Downloader MUST accept an instance of :class:`~.HttpRequest`, not "should accept" it, and it MUST either return an instance of :class:`~.HttpResponse` or raise  a subclass of HttpRequestError.

What do you think about it @BurnzZ ?

Reply:
Good call! Updated the docs in https://github.com/scrapinghub/web-poet/pull/38/commits/b889fb974f935fe0c27902cfdc9cbb2b186fad0f to follow RFC 2119's guidelines.